README.md

```diff
@@ -1,88 +1,55 @@
-<a name="readme-top"></a>
+# Book Motorcycles App
```

Comment:
# Frontend re-review 1

This command is not working:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/f74e61c6-05fc-4fd2-b7a7-59f950a06124)

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/82f4da12-5f92-4331-8796-418716608a73)

If you are not including frontend tests, kindly remove the section. 👍 

- [ ] This link is not working since the last review:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/8afbcb58-8b3f-47ac-83db-5eadf73481ed)

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/248beb13-cccf-420f-a2ec-8c0a0455fcae)


Kindly correct the link which should point to `LICENSE.md`

- [ ] In the details page for mobile view, kindly add more styling so the text does not overflow the screen.

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/716a402b-2e47-4f37-891a-d96d1b51fe98)

Kindly make sure the text fits the screen. 👍 


- [ ] The main "categories" section of the mobile view styling is questionable:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/8103a4d1-a41d-4d8f-af69-0d68dfc89613)

Kindly make it so the text fits. 👍 


- [ ] When trying to create a reservation in the mobile view, the calendar does not fit the container:

![image](https://github.com/naanmohammed/Book-Motorcycles-Backend/assets/57421823/7483ac35-ef89-4094-a798-32856fa60eb6)

Kindly, make it so the container fits. 👍